docs/usage/reserved-ip.md

```diff
@@ -6,7 +6,7 @@
 
 You may have observed that there is a field `excludeIPs` in SpiderIPPool CRD . To some extent, it is also a mechanism for reserving IP, but its main work is not like this. Field `excludeIPs` is more of a **syntax sugar**, so that users can more flexibly define the IP address ranges of the IPPool.
```

Comment:
```suggestion
You may have observed that there is a field `excludeIPs` in SpiderIPPool CRD. To some extent, it is also a mechanism for reserving IPs, but its main function is not like this. Field `excludeIPs` is more of a **syntax sugar**, so that users can more flexibly define the IP address ranges of the IPPool.
```